builder-20/builder.toml

```diff
@@ -137,10 +137,11 @@ version = "0.20.1"
     id = "heroku/eol-warning"
     version = "1.0.0"
 
+# this group is intentionally at the end, because projects in other languages often have e.g. a package.json
```

Comment:
I have no issues with moving Node.js to the end. I think most specific to least specific is a pretty good strategy that the Node.js buildpack uses to resolve package manager selection, so it makes sense here. 

However -- couldn't this also be solved by adding  an optional `heroku/nodejs` to the `heroku/php` group? Basically, solve this the way `heroku/ruby` is?

Reply:
The issue is that for many of these projects, there is no lock file for `package.json`, so the build would fail - a new Laravel framework skeleton project, for example.